membership-attribute-service/app/repositories/MembershipAttributesRepository.scala

```
@@ -21,10 +21,14 @@ class MembershipAttributesRepository @Inject() (dynamo: AmazonDynamoDBScalaMappe
   def updateAttributes(attributes: MembershipAttributes): Future[Unit] =
     if (attributes.tier.isEmpty) {
       logger.debug(s"Delete attributes: $attributes")
-      dynamo.delete(attributes)
+      dynamo.delete(attributes).recover {
+        case t => Logger.error(s"Failed to delete attributes $attributes", t)
```

We want to rethrow the error